installer/pkg/workflow/destroy.go

```diff
@@ -1,5 +1,18 @@
 package workflow
 
+import (
+	"fmt"
+	"path/filepath"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
+)
+
+const (
+	kubeconfigPath = "generated/auth/kubeconfig"
```

Comment:
This is currently being renamed to `kubeconfig-admin` in the next-gen installer (discussion [here][1]).  It would be nice if the generator package stuck this path (or some of it?) into a public variable so we don't have to hard-code it here.

Also, I'm not sure if folks will be running the installer from Windows or not, but I have a slight preference towards:

```go
var (
  kubeconfigPath = filepath.Join("generated", "auth", "kubeconfig")
)
```

[1]: https://github.com/openshift/installer/pull/160#discussion_r213863667